### ISSUE TITLE
JW Player 7.6.1 is not disabled

### ISSUE THREAD

**Issue by alw1976** [Eloston/disable-html5-autoplay#98]
Went to this website and the video played..
http://www.kmbc.com/article/kids-help-get-christmas-trees-to-military-families/8383587

**Comment by ejep1669** [comment]
7.8.6 as well. https://www.fastcompany.com/3066331/innovation-agents/verily-created-a-spoon-that-helps-people-with-movement-disorders-eat-indep

**Comment by brianjmurrell** [comment]
And 7.8.7.

**Comment by brianjmurrell** [comment]
So many open issues about it not working here and there.  Seem this extension is probably too hit-and-miss to be very useful.  :-(

**Comment by Eloston** [comment]
@brianjmurrell Yeah, that's a pretty accurate assessment for the 0.6 series and earlier. The rewrite for version 0.7 (#114) should fix a lot of the underlying issues that are breaking or missing these sites.

**Comment by brianjmurrell** [comment]
@Eloston Looking forward to it then.

**Comment by mrjf** [comment]
JW Player 7.11.2 autoplaying for me here: https://www.digitaltrends.com/mobile/best-camera-phones/
